### PR TITLE
[NO-TICKET] ESLint fixes

### PR DESCRIPTION
**Pull request by nichia**
### Changed
- ESLint fixes

### CODE REVIEW

File: packages/design-system/src/styles/settings/uswds/_uswds-theme-utilities.scss [hunk 1]
```diff
@@ -32,23 +32,14 @@ used by utilities or layout grid
...
-    'widescreen': false,
-  // 1400px
+  'card-lg': false,
+  'mobile': false,
+  'mobile-lg': true,
+  'tablet': true,
+  'tablet-lg': false,
+  'desktop': true,
+  'desktop-lg': false,
+  'widescreen': false,
```
Comment (nichia) [inline packages/design-system/src/styles/settings/uswds/_uswds-theme-utilities.scss:42]: Had to manually removed the size comments as eslint incorrectly fixes this.

File: packages/design-system-docs/src/scripts/components/CodeSnippet.jsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import React from 'react';
 
 const CodeSnippet = function (props) {
   return (
+    /* eslint-disable react/no-danger */
```
Comment (bernardwang) [inline packages/design-system-docs/src/scripts/components/CodeSnippet.jsx:6]: I really like the `eslint-disable-next-line` that you used, can you use that here too?

File: packages/design-system/src/styles/settings/uswds/_uswds-theme-utilities.scss (hunk 1 above)
Comment (bernardwang) [reply packages/design-system/src/styles/settings/uswds/_uswds-theme-utilities.scss:42]: Ah interesting, what does eslint do?

Comment (nichia) [reply packages/design-system/src/styles/settings/uswds/_uswds-theme-utilities.scss:42]: It removed the comments but messed up the object alignment and still shows the lint issue of needing 2 spaces.

**Review by bernardwang — APPROVED** [review]
LGTM!

**Pull request merged**